lib/house.rb

```diff
@@ -34,4 +34,12 @@ def price_per_square_foot
   def rooms_sorted_by_area
     @rooms.sort_by {|room| room.area}
   end
+
```

Comment:
I know there is a much more efficient way to do this but I was drawing a blank at the time so this is the route I went.

Reply:
This works. It's the first one I reached to as well when doing this method. However, you can use `sort` here with a block and it will avoid having to call reverse.